### ISSUE TITLE
Support for css isolation in Blazor Server on m1 Mac

### ISSUE THREAD

**Issue by mrlife** [koenvzeijl/AspNetCore.SassCompiler#87]
**Describe the bug**
Building the project doesn't update the .razor.css file when there are changes to the .razor.scss file.

**To Reproduce**
1. Change a color in a .razor.scss file
2. Build the project
3. Look at the associated .razor.css file for the change
4. See no change

**Expected behavior**
CSS isolation appears to be [supported](https://github.com/koenvzeijl/AspNetCore.SassCompiler/issues/37).

**Desktop (please complete the following information):**
 - OS: macOS 12.3.1 (m1)

**Comment by adiessl** [comment]
What version of `AspNetCore.SassCompiler` are you using, @mrlife? If you are not using the latest version, you should update and test again. Maybe the problem you have is the same I had and fixed in #79.

**Comment by adiessl** [comment]
Yes, I also get errors now when building this project. It seems that this is due to the changes from commit https://github.com/koenvzeijl/AspNetCore.SassCompiler/commit/e51b12e82de86bab9139155c12d75e807d2a85ca.

>I tried the latest: 1.51.0.4.

Just to clarify: you tried the NuGet package version in your actual project or you just tried builing the sample Razor class library?

**Comment by mrlife** [comment]
@adiessl Yes, I did both actually. In my actual project, nothing happens. In the razor class library sample, I get the error.

**Comment by adiessl** [comment]
Have you configured your project according to the [documentation](https://github.com/koenvzeijl/AspNetCore.SassCompiler#configuration), @mrlife?

**Comment by mrlife** [comment]
@adiessl I looked through that, but it wasn't clear how it applies to CSS isolation since those .css files can be anywhere in the project.

**Comment by adiessl** [comment]
Yes. If you did not explicitly configure `AspNetCore.SassCompiler`, the defaults are used. In case of scoped CSS files that means only files in these folders get compiled: `"Views", "Pages", "Shared", "Components"`. If your Razor files are for example in a folder named  `RazorFiles`, the compilation will not include them.

**Comment by mrlife** [comment]
Thanks for clarifying scoped scss. Here is what I have done so far:

1. Add nugget package: `<PackageReference Include="AspNetCore.SassCompiler" Version="1.51.0.4" />`
2. Change a color in scss file at Components/MyComponent.razor.scss
3. Rebuild the project
4. Check for changes in Components/MyComponent.razor.css

I thinks this should work without configuration, if I'm reading it right.

**Comment by adiessl** [comment]
If you delete `Components/MyComponent.razor.css` and build the project, does it get created again?

**Comment by adiessl** [comment]
I created a small demo application: https://github.com/adiessl/SassCompilerDemo

When I build this, the `.razor.scss` file gets compiled. If I change something in it and build again, the change gets reflected in the `.razor.css` file. If you try with this demo, does it work on your side, @mrlife?

**Comment by mrlife** [comment]
@adiessl Yes, this demo is working on my end. I also see it now works in my project as well. I updated my Visual Studio for Mac since I opened this ticket, so I can only assume something changed with that.

Thanks so much for your help! It's great to have a simple solution to compiling scss files.
